website/docs/discover-podman-desktop.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 2
+title: Discover Podman Desktop
+description: Discovering the Podman Desktop application
+keywords: [podman desktop, podman, Discover features]
+tags: [podman-desktop, discover-podman-desktop, features]
+---
+
+# Discover Podman Desktop
+
+Podman Desktop is a community-driven tool that is an alternative to Podman and Docker tools. You can perform a wide range of basic and advanced tasks. From creating a container or pod to setting up a kubernetes cluster to running an AI application to integrating your local tools using extensions, Podman Desktop offers all these features to ease your daily development tasks. But, what makes Podman Desktop different from other similar community-driven tools?
```

Comment:
Podman Desktop is not an alternative to Podman it is a developer tool on top of Podman